akka-actor/src/main/resources/reference.conf

```diff
@@ -81,6 +81,12 @@ akka {
   # such as OutOfMemoryError
   jvm-exit-on-fatal-error = on
 
+  # Akka installs JVM shutdown hooks by default, e.g. in CoordinatedShutdown and Artery.
+  # This property makes it possible to disable all such hooks if the application itself
+  # or a higher level framework such as Play prefers to install the JVM shutdown hook and
```

Comment:
:+1:, let's remember to remove the Play reference when/if they decide not to install their own hooks anymore :)